frame/contracts/src/tests.rs

```diff
@@ -709,7 +735,7 @@ fn instantiate_and_call_and_deposit_event() {
 					phase: Phase::Initialization,
 					event: RuntimeEvent::Balances(pallet_balances::Event::Endowed {
 						account: deposit_account.clone(),
-						free_balance: 131,
+						free_balance: 132,
```

Comment:
Where is this extra 1 coming from?

Reply:
I think this is because `bytes_added: self.encoded_size()` grew by 1 because of the new delegate_dependencies field